src/packages/galaxy/galaxy.py

```diff
@@ -10,11 +10,12 @@
 See LICENSE.md
 """
 
+import asyncio
 from html import escape
 import json
 from typing import Dict, List, Optional, Tuple
 
-import aiohttp
+from aiohttp import ClientError, ClientSession, ClientTimeout
```

Comment:
[style] [non-blocking] For the sake of clarity on where these exceptions come from at a glance, i would prefer you to just import `aiohttp` here

Reply:
704dd81